pnpm-lock.yaml

```diff
@@ -29903,6 +29920,22 @@ snapshots:
       - supports-color
       - utf-8-validate
 
+  '@fluid-tools/version-tools@0.58.1(@types/node@18.19.86)':
```

Comment:
This "duplicate" (and similar for `@fluidframework/build-tools@0.58.1(@types/node@18.19.86)` below, and some other packages) that uses `@types/node@18.19.86` instead of `18.19.67` which is what the lockfile resolves it to, are interesting... I recall we ran into something like this before and couldn't figure out where these were coming from. IIRC it wasn't global installs of those latest @types/node@18 packages?

Reply:
Pff, the same thing happened to me on a different update 😆 .

Reply:
I updated the 2 big lockfiles, but dedupe made quite a few unrelated changes. I'm not sure that's better. We should probably do a separate dedupe PR to get our locks in a better place...

Reply:
https://github.com/microsoft/FluidFramework/pull/25378